index.js

```diff
@@ -28,7 +29,13 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let allMovies = []
+  for( let i=0 ; i < movies.length ; i++){
```

Comment:
Since we are not using `i` for anything besides accessing an element in our `movies` array, this should be a `for...of` loop.